src/main/java/com/dynatrace/jenkins/dashboard/TABuildWrapper.java

```diff
@@ -170,16 +193,17 @@ private void setupBuildVariables(AbstractBuild build, String serverUrl) {
 		if (StringUtils.isNotEmpty(serverUrl)) {
 			parameters.add(new StringParameterValue(BuildVarKeys.BUILD_VAR_KEY_GLOBAL_SERVER_URL, serverUrl));
 		}
-		if (StringUtils.isNotEmpty(globalConfig.username)) {
+		if (globalConfig != null && StringUtils.isNotEmpty(globalConfig.username)) {
 			parameters.add(new StringParameterValue(BuildVarKeys.BUILD_VAR_KEY_GLOBAL_USERNAME, globalConfig.username));
 		}
-		if (StringUtils.isNotEmpty(globalConfig.password)) {
+		if (globalConfig != null && StringUtils.isNotEmpty(globalConfig.password)) {
 			parameters.add(new PasswordParameterValue(BuildVarKeys.BUILD_VAR_KEY_GLOBAL_PASSWORD, globalConfig.password));
 		}
 		Utils.updateBuildVariables(build, parameters);
 	}
 
 	@Extension
+	@Symbol("appMonBuildEnvironment")/**is a function name in pipeline script*/
```

Comment:
If it's non-javadoc comment, use `/* */`
If it's javadoc comment, write it in new line, above annotations.